### PR TITLE
Importing GitHub Repository (Action): AzViz (Azure Visualizer) action

### PR DESCRIPTION
**Pull request by emvaldes**
This PR Importing GitHub Repository (Action): AzViz (Azure Visualizer) action
**This PR does not suggest a fix for a currently exploitable issue.**

### Process
- [x] Are there licensing issues with any new dependencies introduced? No
- [x] Includes a summary of what a code reviewer should test/verify? No
- [x] Updated the release notes? No
- [x] Database changes are submitted as a separate PR? No
- [x] DevOps team has been notified if PR requires ops support? We are the ones performing these changes.

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints? No
- Does this PR include changes in authentication and/or authorization of existing endpoints? No
- Does this change introduce new dependencies that need vetting? No
- Does this change require changes to our infrastructure? No
- Does logging contain sensitive data? No
- Does this PR include or remove any sensitive information itself? No

- What are the potential security threats and mitigations? N/A
- Have you ensured logging does not contain sensitive data? No
- Have you received any additional approvals needed for this change? No

### CODE REVIEW

File: .github/actions/azviz/.gitignore
Comment (devopsmatt) [inline .github/actions/azviz/.gitignore:1]: This shouldn't be needed here.

**Review by devopsmatt — COMMENTED** [review]
Minor changes requested.

File: .github/actions/azviz/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+jobs:
+  generate-viz:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Login to Azure
+          uses: azure/login@v1
+          with:
+            creds: ${{ secrets.SERVICE_PRINCIPAL_CREDS }}
+            enable-AzPSSession: true
+        - uses: CDCgov/prime-reportstream/.github/actions/azviz@663e24299a6336f1ff8dbddadfac1ba5d462f731aaa
```
Comment (devopsmatt) [inline .github/actions/azviz/README.md:77]: We should version pin, not SHA.

Comment (emvaldes) [reply .github/actions/azviz/README.md:77]: I totally agree. Let me ask you something:
Which version should we use? This is not an actual GitHub repo but rather an imported repo inside another repo and not as a sub-repo. So, unless it's decided to define an abstracted version-id, there is no version.
This is a README.md file and this change is intended to keep track of what hardcoded hash commit is defined within the CI/CD pipeline in the project's GitHub Action.

File: .github/actions/azviz/.gitignore
Comment (emvaldes) [reply .github/actions/azviz/.gitignore:1]: There is no technical reason for why it should not be there as it's something we should standardize in all our imports to have a mechanism to define what type of files should not be committed. Nevertheless, if you do not want it there, I will remove it.

**Review by devopsmatt — APPROVED** [review]

**Pull request merged**